openassessment/templates/openassessmentblock/staff_area/oa_staff_assessment.html

```diff
@@ -0,0 +1,52 @@
+{% load tz %}
+{% load i18n %}
+{% spaceless %}
```

Comment:
https://code.djangoproject.com/ticket/1067

Useful for being about to put commands on multiple lines.